src/main/java/core/basesyntax/services/FruitMap.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.services;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitMap {
+    private static final String SYMBOL = ",";

```

Comment:
If you name this symbol more specifically, the code will be more readable. Something like REGEX_COMMA